guacamole/src/main/java/org/apache/guacamole/tunnel/TunnelRequestService.java

```diff
@@ -226,7 +300,7 @@ protected GuacamoleTunnel createConnectedTunnel(UserContext context,
      * @throws GuacamoleException
      *     If an error occurs while obtaining the tunnel.
      */
-    protected GuacamoleTunnel createAssociatedTunnel(GuacamoleTunnel tunnel,
+    protected GuacamoleTunnel createAssociatedTunnel(final GuacamoleTunnel tunnel,
```

Comment:
It gets referenced in the anonymous inner `UserTunnel` type. Previously, it wasn't referenced, so this didn't need to be final.